examples/plot_diabetes_variable_importance_example_v2.py

```diff
@@ -0,0 +1,104 @@
+# %%
```

Comment:
Please find a better name for the example.
I believe that this example is meant to replace the other one ?